substratevm/src/com.oracle.svm.core.posix/src/com/oracle/svm/core/posix/JavaNetNetUtil.java

```diff
@@ -223,6 +223,114 @@ static InetAddress NET_SockaddrToInetAddress(Socket.sockaddr him, CIntPointer po
         return iaObj;
     }
 
+    // 255 JNIEXPORT jint JNICALL
+    // 256 NET_SockaddrEqualsInetAddress(JNIEnv *env, struct sockaddr *him, jobject iaObj)
+    static boolean NET_SockaddrEqualsInetAddress(Socket.sockaddr him, InetAddress iaObj) {
+        // 258 jint family = AF_INET;
+        int family = Socket.AF_INET();
+
+        int sa_family = Socket.AF_INET();
```

Comment:
This variable appears to be unused.  Probably you thought you would need it to untangle the mess below.